suite-common/token-definitions/package.json

```diff
@@ -23,6 +23,7 @@
         "jws": "^4.0.0"
     },
     "devDependencies": {
+        "@trezor/blockchain-link-utils": "workspace:*",
```

Comment:
This is on purpose? I think i should be inside `dependencies`

Reply:
Why, it is used in scripts, not in the app. I will move it but there has to be something wrong with configuration

Reply:
it is used in the same place as `ajv` and that dep can be in devDependencies so i dont know